tasks/src/zero-shot-image-classification/data.ts

```diff
@@ -0,0 +1,48 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "",
+			id: "",
+		},
+	],
+	demo: {
+		inputs: [],
+		outputs: [],
+	},
+	metrics: [
+		{
+			description:
+				"Computes the number of times the correct label appears in top K labels predicted",
+			id: "top-K accuracy",
+		},
+	],
+	models: [
+		{
+			// TO DO: write description
+			description:
+				"Robust image classification model trained on publicly available image-caption data.",
+			id: "openai/clip-vit-large-patch14",
+		},
+		{
+			// TO DO: write description
+			description:
```

Comment:
Can you briefly explain what are the difference between CLIP checkpoints that you've put?

Reply:
pls let me know if the desc here is ok...afaik the differences are due to the number of patches used in the versions one uses 14 and the other uses 16 ?

Reply:
Yes it's just I thought we could add more context, can we somehow put it to simpler terms in the eyes of a software developer, how does it affect the output that there are more patches? Which one has bigger number of parameters, does it affect the performance & latency?